package/rpm-config-SUSE.spec

```diff
@@ -25,16 +25,14 @@ License:        GPL-2.0-or-later
 Group:          System/Packages
 URL:            https://github.com/openSUSE/rpm-config-SUSE
 Source:         %{name}-%{version}.tar.gz
-
+#!BuildIgnore:  rpm-config-SUSE
 # RPM owns the directories we need
 Requires:       rpm
-
 BuildArch:      noarch
-#!BuildIgnore:  rpm-config-SUSE
 
 %description
-This package contains the RPM configuration data for the SUSE Linux
-distribution family.
+This package contains the RPM configuration data for the SUSE and
+openSUSE distribution families.
```

Comment:
The entire family is known as the "SUSE Linux" family, broken up into SLE and openSUSE. Does it _really_ need to made more explicit?

Reply:
do you have any pointer for that hypothesis? for me SUSE Linux is simply the old name of openSUSE, and both openSUSE and SUSE are their own families. 

I think it makes sense to mention it that way because the string "openSUSE" might be the key word that people look for, as I had already people asking where is rpm-config-openSUSE.